packages/dashboard/src/layout/GoldenLayout.scss

```diff
@@ -50,6 +50,7 @@ $lm-splitter-active-color: $gray-300;
   border: 15px solid rgba($black, 0.25);
```

Comment:
Oh, hey this line is actually adding an underlay already, and maybe should just be turned up?

Try 0.85?

Reply:
Ooh yes you're right good catch! I forgot it's kind of reversed. To decrease the visibility of the background I gotta increase the opacity of this border